src/components/CardList/Card/Card.tsx

```diff
@@ -0,0 +1,48 @@
+import * as S from "./Card.style";
+import { useState } from "react";
+import { getTimeDifference } from 'utils/getTimeDifference';
+import { formatDate } from 'utils/formatDate';
+import starIcon from "assets/images/starIcon.svg";
+import kebabIcon from "assets/images/kebabIcon.svg";
+import noImg from "assets/images/noImg.svg";
+import SelectMenu from 'components/SelectMenu/SelectMenu';
+
+
+function Card({ item }) {
+  const [showSelectMenu, setShowSelectMenu] = useState(false);
```

Comment:
요 경우엔 타입스크립트 적용하면 이런 형태가 될 수 있겠죠? ㅎㅎ
```suggestion
  const [showSelectMenu, setShowSelectMenu] = useState<boolean>(false);
```